test/performance/odp_sched_pktio.c

```diff
@@ -1337,10 +1336,6 @@ static int start_timers(test_global_t *test_global)
 	if (test_global->opt.timeout_us == 0)
 		return 0;
 
-	/* Delay the first timeout so that workers have time to startup */
-	timeout_tick += odp_timer_ns_to_tick(test_global->timer.timer_pool,
-					     TIMEOUT_OFFSET_NS);
-
```

Comment:
After removing the offset, timers can trigger too early if `start_workers()` or `start_pktios()` calls are slow. Perhaps the simplest solution to mitigate this potential issue would be to reorder the functions calls in `main()`, so that `start_timers()` is the last called function before `odp_barrier_wait()` and `start_pktios()` is moved up before `start_timers()`.

Reply:
ack, I will reorder the calls in v2. Thanks!